web/src/js/flow/utils.js

```diff
@@ -49,7 +49,14 @@ export var MessageUtils = {
         } else if (message === flow.response) {
             message = "response";
         }
-        return `/flows/${flow.id}/${message}/content` +  (view ? `/${view}` : '');
+        if (global.MITMWEB_STATIC) {
```

Comment:
Maybe we can just add `(?:\.json)?` to our routes, e.g. [here](https://github.com/mitmproxy/mitmproxy/blob/96c8c75ce6b8c656e5f03312479c32082d5a89b7/mitmproxy/tools/web/app.py#L472):

```python
(r"/flows(?:\.json)?", Flows),
```
Thoughts? 😃